src/controlplane/Cargo.toml

```diff
@@ -18,5 +18,5 @@ tracing = "0.1.19"
 # Fluvio dependencies
 fluvio-types = { path = "../types", version = "0.2.0" }
```

Comment:
need bump up crate

Reply:
This has already been bumped and not published, so I think we can ride on this version